### ISSUE TITLE
Keras crash on backend CNTK when using Conv2DTranspose with padding: "same" and kernel size large than input dimension

### ISSUE THREAD

**Issue by maybeLee** [keras-team/keras#15173]
**Describe the problem**.

Describe the problem clearly here. Be sure to convey here why it's a bug in Keras or why the requested feature is needed.

**Describe the current behavior**.
Keras will directly crash when using Conv2DTranspose API based on CNTK by giving some specific inputs.
Specifically, when we set the kernel size larger than the input dimension and set the padding dimension to be "same", this crash will happen even when we randomly generate the input. However, when I switch Keras to a different backend, such a problem will not exist.

**Describe the expected behavior**.
Keras will not crash on the above situation

**Standalone code to reproduce the issue**.
```
import os
import argparse
import sys
import warnings
parse = argparse.ArgumentParser()
parse.add_argument("--backend", type=str,default="cntk", help="the name of backend")
flags, _ = parse.parse_known_args(sys.argv[1:])
os.environ["KERAS_BACKEND"]=flags.backend
bk = flags.backend
if bk == "cntk":
  from cntk.device import try_set_default_device,gpu
  try_set_default_device(gpu(0))
  import cntk as ck
import keras
from keras import initializers, layers
import numpy as np
warnings.filterwarnings("ignore", category=DeprecationWarning)
warnings.filterwarnings("ignore", category=UserWarning)
model_1 = keras.models.Sequential()
model_1.add(layers.Conv2DTranspose(1,(3,3), padding="same",  data_format="channels_first"))
x = np.random.rand(100,28,28,1)
pred = model_1.predict(x)
```

**Comment by jvishnuvardhan** [comment]
@maybeLee Starting `TF2.6`, we are supporting only TF backend. Sorry for the inconvenience.

Please post it in Stackoverflow where there is a large community to support and learn from each other's issues. If you face any Bugs and performance related issues with recent TF/Keras versions then please post them here. Thanks!
